samples/transactional-session/aspnetcore-webapi/TransactionalSession_1/WebApplication/WebApplication.csproj

```
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <LangVersion>7.3</LangVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="NServiceBus.Extensions.Hosting" Version="1.1.0" />
+    <PackageReference Include="NServiceBus.Persistence.Sql.TransactionalSession" Version="6.6.0-rc.1" />
+    <PackageReference Include="NServiceBus.Persistence.Sql" Version="6.6.0-rc.1" />
```

```suggestion
    <PackageReference Include="NServiceBus.Persistence.Sql.TransactionalSession" Version="6.6.0" />
    <PackageReference Include="NServiceBus.Persistence.Sql" Version="6.6.0" />
```

or

```suggestion
    <PackageReference Include="NServiceBus.Persistence.Sql.TransactionalSession" Version="6.6.0" />
```

is actually enough due to the transitive dependency to the specific version